### PR TITLE
Feature: Container filter

### PR DESCRIPTION
**Pull request by Hundraw**
Added a feature to filter the components that can enter a container.

### CODE REVIEW

File: demo/src/assets/data.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ const networkDefinition = new ComponentDefinition({
   icon: 'network',
   model: 'DefaultContainer',
   parentTypes: [],
```
Comment (Zorin95670) [inline demo/src/assets/data.js:42]: you miss the parentTypes

File: src/draw/DefaultDrawer.js [hunk 2]
```diff
@@ -205,47 +205,54 @@ class DefaultDrawer {
...
-        currentComponent = this.d3.select(`#${event.relatedTarget.id}`);
-        invalidDropzones = [];
-        if (currentComponent.datum().definition.isContainer) {
-          this.__getChildrenContainer(currentComponent.datum().children, invalidDropzones);
-        }
-      },
-      ondrop: (event) => {
-        if (!invalidDropzones.includes(event.target.id)) {
-          const dropzone = this.d3.select(`#${event.target.id}`).datum();
+    components.forEach((component) => {
```
Comment (Zorin95670) [inline src/draw/DefaultDrawer.js:208]: according to sonar, you have to rewrite this to remove complexity

File: changelog.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Demo of `DefaultDrawer`.
 - Add feature to drag and drop component into container components in `DefaultDrawer`.
 - Install `interactjs` library.
+- Added a feature to filter the components that can enter a container.
```
Comment (Zorin95670) [inline changelog.md:15]: don't forget the new attribute

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: demo/src/assets/data.js (hunk 1 above)
Comment (Hundraw) [reply demo/src/assets/data.js]: Done

File: changelog.md (hunk 3 above)
Comment (Hundraw) [reply changelog.md:15]: Done

File: src/draw/DefaultDrawer.js (hunk 2 above)
Comment (Hundraw) [reply src/draw/DefaultDrawer.js]: Done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**